contrib/glfw/template.py

```diff
@@ -0,0 +1,27 @@
+pkgname = "glfw"
+pkgver = "3.3.9"
```

Comment:
you downgraded this again by accident, though of course you'll have to fix the 3.4 build